README.md

```diff
@@ -97,28 +113,30 @@ If CallKit has been setup, this will make a phone call to Tom's device.
 From the remote device, when the call is ready to be answered, you can join the streem with the following:
 
 ```swift
-    let state = StreemStateBuilder()
-        .with(myRole: .REMOTE_PRO)
-        .with(remoteUserId: fromUserId, andRole: .LOCAL_CUSTOMER)
-        .joining(streemId: streemId, fromCallId: callId)
-        .build()
-
-    Streem.openStreem(state)
+    let remoteUser = StreemUser(name: userName, id: userId)
+    Streem.sharedInstance.startRemoteStreem(
```

Comment:
This method is currently `internal`, and needs to be made `public`.

Reply:
I wonder if we really need to have this on the public api.  The "answering" of the call will always happen within our code, so it could probably remain internal, and we can just remove the documentation of it.